src/features/verification/VerificationResultsDialog.module.css

```
@@ -0,0 +1,191 @@
+.technicalDetail small {
+  display: block;
+  margin-top: 4px;
+}
+
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)